static/app/views/ddm/metricsExplorer.tsx

```diff
@@ -326,15 +334,43 @@ function MetricsExplorerDisplay({displayType, ...metricsDataProps}: DisplayProps
     );
   }
 
+  // TODO(ddm): we should move this into the useMetricsData hook
   const sorted = sortData(data);
+  const unit = getUnitFromMRI(Object.keys(data.groups[0].series)[0]); // this assumes that all series have the same unit
```

Comment:
```suggestion
  const unit = getUnitFromMRI(Object.keys(data.groups[0]?.series ?? {})[0]); // this assumes that all series have the same unit
```
I changed this recently ^